### PR TITLE
added check filename case in win32 FileUtils

### PR DESCRIPTION
**Pull request by qiutaoleo**
make win32 check filename case like linux

### CODE REVIEW

**Comment by darkdukey** [comment]
[ci rebuild]

File: cocos/platform/win32/CCFileUtils-win32.cpp [hunk 1]
```diff
@@ -129,6 +129,48 @@ bool FileUtilsWin32::isAbsolutePath(const std::string& strPath) const
     return false;
 }
 
+static bool checkFileName(const std::string& fullPath, const std::string& filename)
+{
+	//效验文件大小写
```
Comment (minggo) [inline cocos/platform/win32/CCFileUtils-win32.cpp:134]: Please don't use Chinese?

File: cocos/platform/win32/CCFileUtils-win32.cpp [hunk 2]
```diff
@@ -129,6 +129,48 @@ bool FileUtilsWin32::isAbsolutePath(const std::string& strPath) const
...
+			break;
+
+		do
+		{
+			tmpPath = tmpPath.substr(0, tmpPath.rfind("/"));
+		} while (tmpPath.back() == '.');
+	}
+	return true;
+}
+
```
Comment (minggo) [inline cocos/platform/win32/CCFileUtils-win32.cpp:173]: Please replace tab with 4 spaces.

**Comment by walzer** [comment]
@minggo Please help @qiutaoleo to fix these tiny mistakes, and merge this pull request. Don't be blocked on these small things like tab or Chinese comments.

**Comment by minggo** [comment]
Ok, i will help to fix it.
But it is better to let our developers to know the coding styles.

**Comment by minggo** [comment]
Done in #11611.

**Comment by qiutaoleo** [comment]
thanks

**Pull request closed without merging**